enforcer.go

```diff
@@ -440,6 +448,16 @@ func (e *Enforcer) BuildIncrementalRoleLinks(op model.PolicyOp, ptype string, ru
 	return e.model.BuildIncrementalRoleLinks(e.rmMap, op, "g", ptype, rules)
 }
 
+// NewTypeContext Create a default structure based on the suffix
+func NewTypeContext(suffix string) TypeContext {
```

Comment:
I think the suffix is unnecessary.

Reply:
If someone use this method,it means he needs more type such as ``r2,p2``,so I think the ``suffix`` is more convenient.So as not to need to assign one by one